src/kraft.erl

```diff
@@ -15,24 +23,45 @@
 
 %--- Types ---------------------------------------------------------------------
 
--export_type([status/0]).
--export_type([headers/0]).
--export_type([params/0]).
--export_type([body/0]).
--export_type([response/0]).
--export_type([conn/0]).
+-export_type([
+    conn/0,
+    status/0,
+    params/0,
+    headers/0,
+    body/0,
+    response/0,
+    route_def/0
+]).
 
--type status() :: kraft_conn:status().
+-type body() :: kraft_json:body_json() | cowboy_req:resp_body().
+-type conn() :: kraft_conn:conn().
 -type headers() :: #{binary() => iolist()}.
 -type params() :: kraft_conn:params().
--type body() :: kraft_json:body_json() | cowboy_req:resp_body().
 -type response() :: kraft_handler:response().
--type conn() :: kraft_conn:conn().
+-type route_def() :: kraft_instance:route_def().
+-type status() :: kraft_conn:status().
 
 %--- API -----------------------------------------------------------------------
 
+-doc #{equiv => start(Opts, [])}.
 start(Opts) -> start(Opts, []).
 
+-doc """
+Starts a web server with the specified `Opts` options and `Routes` route definitions.
```

Comment:
The first line summary should be shorter and can avoid obvious explanations such as describing the variable names.